src/main/app/forms/validation/validationErrors.ts

```diff
@@ -7,4 +7,11 @@ export class ValidationErrors {
   static readonly VALID_OWED_AMOUNT_REQUIRED: string = 'Enter a valid amount owed'
   static readonly AMOUNT_REQUIRED: string = 'Enter an amount'
   static readonly AMOUNT_INVALID_DECIMALS: string = 'Enter valid amount, maximum two decimal places'
+  static readonly OPTION_REQUIRED: string = 'Choose option'
```

Comment:
On line 6 there is `static readonly SELECT_AN_OPTION: string = 'Select an option'` -- I'm pretty sure they relate to the same type of validation. Can you please contact the content team and check which one should be used?

Reply:
In the story the acceptance criteria expects Error message - "Please select when you will pay"

Reply:
In that case use the message defined in story description. But please make it specific to repayments, not a generic one.